client/app/account/billing/components/filters/renewFrequence.js

```diff
@@ -1,23 +1,17 @@
-angular.module("Billing.filters").filter("renewFrequence", ($translate) => function (period) {
-    let txtFrequence = null;
-    switch (period) {
-    case 1:
-        txtFrequence = $translate.instant("autorenew_service_renew_month");
-        break;
-    case 3:
-    case 6:
-        txtFrequence = $translate.instant("autorenew_service_renew_frequency_value", [period]);
-        break;
-    case 12:
-        txtFrequence = $translate.instant("autorenew_service_renew_year");
-        break;
-    default:
-        if (period > 12) {
-            txtFrequence = $translate.instant("autorenew_service_renew_years_frequency_value", [period / 12]);
-        } else {
-            txtFrequence = $translate.instant("autorenew_service_renew_frequency_value", [period]);
-        }
-        break;
-    }
-    return txtFrequence;
-});
+angular
+    .module("Billing.filters")
+    .filter("renewFrequence", ($translate) =>
+        (period) => {
+            switch (period) {
+            case 1:
+                return $translate.instant("autorenew_service_renew_month");
+            case 12:
+                return $translate.instant("autorenew_service_renew_year");
+            default:
+                if (period > 12) {
+                    return $translate.instant("autorenew_service_renew_years_frequency_value", { t0: period / 12 });
```

Comment:
`period` could be more than 12, so it could be 18,19 or 20 no ? 
If 20 => 20 / 12 = 1.4166667, no need to handle multiple decimals ?

Reply:
Excellent remark. I can't know for sure, but I guess we are only supposed to send `input % 12 === 0`.

I'll throw an error if someone tries to send something that is `input % 12 !== 0`